manifest/test/acceptance/testdata/ComputedFields/computed_deployment.tf

```
@@ -0,0 +1,52 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+resource "kubernetes_manifest" "deployment_resource_diff" {
+computed_fields = ["spec.template.spec.containers[0].resources.limits"]
```

Let's test specifically for the case that was causing the initial issue (keys in a map). 
```suggestion
computed_fields = ["spec.template.spec.containers[0].resources.limits[\"cpu\"]"]
```